js/lib/tree.js

```diff
@@ -298,7 +300,7 @@ var TreeView = widgets.DOMWidgetView.extend({
                 }
             }
         ).bind(
-            "before_open.jstree", (evt, data) => {
+            "after_open.jstree", (evt, data) => {
```

Comment:
This should fix the following glitch:

### Before (the animation doesn't have time to complete on open)

![ipysheet-before](https://user-images.githubusercontent.com/591645/54319166-fb28e000-45e8-11e9-8fd7-beb6f218831f.gif)

### After

![ipysheet-after](https://user-images.githubusercontent.com/591645/54319182-05e37500-45e9-11e9-80b8-264e98f4cb4b.gif)